### DISCUSSION TITLE
FedNot's response (eDoc eDeed)

### DISCUSSION THREAD

**Discussion by sharethewisdom** [roelderickx/connective-plugin-linux#21] [General]
So on 11/07/2023 me and my sister had an online meeting with a solicitor who had to read aloud a thing after which we were to sign a notary thing. I asked the secretary beforehand how signatures would be realized online and if I needed anything in particular.

She did not reply, so I was left in a meeting on the Belgian FedNot platform where I wasted everyone's precious time trying to circumvent the flagrant disregard from Connective to the Linux user. In the end, my sister had driven 20km to my home to bring me a Windows laptop: This was faster and easier than setting up a Windows virtual machine on the hardware I had on hand at the time.

Anyway. I expressed my concern to FedNot and got a response:

> Sinds 12/01/2024, werd de rechtstreekse toegang tot connective platform afgesloten en kan u gebruik maken van eDoc en eDeed voor het digitaal ondertekenen van authentieke en niet authentieke akten.

_Since 12/01/2024, direct access to the connective platform was removed, and you can make use of eDoc and eDeed for digitally signing both authentic and non-authentic documents._

I'm not signing anything soon, but I'd like to know if one can get by with Linux for the eDoc or eDeed stuff. Please chime in if you've used it, or if you know what it is or how it works. Thanks.

**Comment by roelderickx** [comment]
Hi,

It is the first time I hear about this fednot application. Last time I had to sign a notarial deed I was obliged to use itsme, which is impossible without smartphone. Their best suggestion in the end was using the smartphone of a family member or friend :-/

I also don't see the point of installing itsme: you need to sign a hype document about itsme using the connective plugin first to be able to sign anything with itsme. That's simply ridiculous. And from a cryptographic point of view it is exactly the same as using an e-id card, except that your data is stored on two servers in stead of one when using itsme.

But ok, back to eDoc / eDeed. I found article 4 in chapter II of their [terms and conditions](https://login.fednot.be/gebruiksvoorwaarden_e-notariaat_5.0.pdf):
> Elke Gebruiker kan met zijn persoonlijke Identificatiekaart en PIN-code:
> - met zekerheid geïdentificeerd en geauthenticeerd worden en op die manier toegang
krijgen tot de Portaalsite van het e-notariaat en haar Applicaties;
> - documenten elektronisch ondertekenen wanneer hij gebruikmaakt van zijn eID-kaart
of itsme.

Yes. That's any user, even those without a smartphone running Linux. Although they do not specify eDoc or eDeed in the whole document. It is notoriously obscure if you ask me, how do they plan user adoption then?

**Reply by sharethewisdom** [reply]
Thanks. Also note:

> Artikel 22. De Gebruiker zorgt ervoor dat hij beschikt over apparatuur die aangepast is aan het gebruik van de diensten van het e-notariaat.

The terms and conditions date from 2019, and it does not mention Connective stuff nor eDoc / eDeed stuff. I can't find the eDoc / eDeed stuff anywhere. This is exhausting and disappointing.

**Reply by roelderickx** [reply]
That article is far too vague to be legally binding, it allows you to use a commodore 64 or even a coffee machine since "adapted equipment" is up to the reader of the article to interprete.
I can understand when a plumber for example makes this kind of mistakes in their conditions, but we're speaking about notaries with a legal background. Painful.
